offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/product/MaintainProductsView.groovy

```diff
@@ -58,10 +58,10 @@ class MaintainProductsView extends FormLayout {
         this.createProductView = createProductView
         this.copyProductView = copyProductView
         setupTitle()
-        setupPanel()
         createButtons()
         setupGrid()
         setupOverviewLayout()
+        setupPanel()
```

Comment:
Why did you move this method? Was there a reason except for alphabetical order? (which I love) If the ordering of the methods was the reason, please consider moving `setupTitle` as well.

Reply:
Its because it also affects the order in which the components are added to the view and therefore affects the position of the panel

Reply:
I see. Maybe document the fact that these methods have the side effect of adding components directly to the MaintainProductsView. In other views, this is solved by the incapsulation of creating objects in the `initLayout` and filling / binding them to data later on.

Reply:
I don't think that this is required. It's quite easy to see from the code. I would rather opt for refactoring. You could create an issue with your suggestion. It makes sense to have the same underlying structure for all of our views.